packages/engine/src/datastore/storage/markers.rs

```diff
@@ -1,3 +1,24 @@
+//! Each shared memory segment contains four buffers. At the start
+//! of the segment, before the first buffer, are eight numbers called
+//! "markers" -- for each buffer, the offset (i.e. start index in bytes)
+//! and size (i.e. number of bytes) of the buffer.
```

Comment:
This documentation will also need to be updated if we make the metaversion / memory version a fifth marker.

Reply:
The start index is from the start of the segment, e.g. `a_start` is always 64.

Reply:
I updated the documentation as proposed by @Alfred-Mountfield. I wouldn't explicitly state, that the start index of the schema buffer is always at `64` because as soon as we add more markers, this becomes invalid. While we could update the documentation at that point, it's likely to forget it. Also one could accidentally rely on the value of `64` bytes hardcoded, which should be avoided.